gems/pending/appliance_console.rb

```diff
@@ -356,18 +361,21 @@ module ApplianceConsole
         when RESTORE_LOCAL
           validate = ->(a) { File.exist?(a) }
           uri = just_ask("location of the local restore file", DB_RESTORE_FILE, validate, "file that exists")
-          task_with_opts = "evm:db:restore:local -- --local-file '#{uri}'"
+          task = "evm:db:restore:local"
+          task_opts = %W(-- --local-file '#{uri}')
 
         when RESTORE_NFS
           uri = ask_for_uri("location of the remote backup file\nExample: #{sample_url('nfs')})", "nfs")
-          task_with_opts = "evm:db:restore:remote -- --uri '#{uri}'"
+          task = "evm:db:restore:remote"
+          task_opts = %W(-- --uri '#{uri}')
 
         when RESTORE_SMB
           uri = ask_for_uri("location of the remote backup file\nExample: #{sample_url('smb')}", "smb")
           user = just_ask("username with access to this file.\nExample: 'mydomain.com/user'")
           pass = ask_for_password("password for #{user}")
 
-          task_with_opts = "evm:db:restore:remote -- --uri '#{uri}' --uri-username '#{user}' --uri-password '#{pass}'"
+          task = "evm:db:restore:remote"
+          task_opts = %W(-- --uri '#{uri}' --uri-username '#{user}' --uri-password '#{pass}')
```

Comment:
Or just don't give the "--" at all here and default it directly in the `.rake` method.